app/javascript/CommentSubscription/CommentSubscription.jsx

```diff
@@ -123,6 +123,7 @@ export class CommentSubscription extends Component {
           </Button>
           {subscribed && (
             <Button
+              data-testid="subscription-settings"
```

Comment:
`data-testid` is typically used for DOM elements you have trouble getting a reference to. Also,  we should not rely on CSS class names or element IDs as they can change, whereas the `data-testid` would not as it is specific to testing only. This is also a common practice in end to end testing. For more information check out, [Making your UI tests resilient to change](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).